File: KetelTest/KetelTest/Models/NetInputModels/UserSave.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KetelTest.Models.NetInputModels
{
    public class UserSave
    {
        public int ID { get; set; }
        public string Nickname { get; set; }
        public bool Sex { get; set; }
        public int Age { get; set; }
    }
}

File: TestKetel/app/src/main/java/com/example/zimma/testketel/Models/ModelsInteract/UserSave.java
package com.example.zimma.testketel.Models.ModelsInteract;

public class UserSave {

    private int ID;
    private String Nickname;
    private int Age;
    private boolean Sex;

    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public String getNickname() {
        return Nickname;
    }

    public void setNickname(String nickname) {
        Nickname = nickname;
    }

    public int getAge() {
        return Age;
    }

    public void setAge(int age) {
        Age = age;
    }

    public boolean isSex() {
        return Sex;
    }

    public void setSex(boolean sex) {
        Sex = sex;
    }

    public UserSave(int ID, String nickname, int age, boolean sex) {
        this.ID = ID;
        Nickname = nickname;
        Age = age;
        Sex = sex;
    }
}
